src/rdbms/azext_rdbms/commands.py

```diff
@@ -35,124 +25,10 @@ def load_command_table(self, _):
         client_factory=cf_postgres_servers
     )
 
-    mysql_firewall_rule_sdk = CliCommandType(
-        operations_tmpl='azext_rdbms.mysql.operations.firewall_rules_operations#FirewallRulesOperations.{}',
-        client_arg_name='self',
-        client_factory=cf_mysql_firewall_rules
-    )
-
-    postgres_firewall_rule_sdk = CliCommandType(
-        operations_tmpl='azext_rdbms.postgresql.operations.firewall_rules_operations#FirewallRulesOperations.{}',
-        client_arg_name='self',
-        client_factory=cf_postgres_firewall_rules
-    )
-
-    mysql_config_sdk = CliCommandType(
-        operations_tmpl='azext_rdbms.mysql.operations.configurations_operations#ConfigurationsOperations.{}',
-        client_arg_name='self',
-        client_factory=cf_mysql_config
-    )
-
-    postgres_config_sdk = CliCommandType(
-        operations_tmpl='azext_rdbms.postgresql.operations.configurations_operations#ConfigurationsOperations.{}',
-        client_arg_name='self',
-        client_factory=cf_postgres_config
-    )
-
-    mysql_log_sdk = CliCommandType(
-        operations_tmpl='azext_rdbms.mysql.operations.log_files_operations#LogFilesOperations.{}',
-        client_arg_name='self',
-        client_factory=cf_mysql_log
-    )
-
-    postgres_log_sdk = CliCommandType(
-        operations_tmpl='azext_rdbms.postgresql.operations.log_files_operations#LogFilesOperations.{}',
-        client_arg_name='self',
-        client_factory=cf_postgres_log
-    )
-
-    mysql_db_sdk = CliCommandType(
-        operations_tmpl='azext_rdbms.mysql.operations.databases_operations#DatabasesOperations.{}',
-        client_arg_name='self',
-        client_factory=cf_mysql_db
-    )
-
-    postgres_db_sdk = CliCommandType(
-        operations_tmpl='azext_rdbms.postgresql.operations.databases_operations#DatabasesOperations.{}',
-        client_arg_name='self',
-        client_factory=cf_postgres_db
-    )
-
     with self.command_group('mysql server', mysql_servers_sdk, client_factory=cf_mysql_servers) as g:
-        g.command('create', 'create')
-        g.custom_command('restore', '_server_restore', no_wait_param='no_wait')
-        g.command('delete', 'delete', confirmation=True)
-        g.command('show', 'get')
-        g.custom_command('list', '_server_list_custom_func')
-        g.generic_update_command('update',
-                                 getter_name='_server_update_get', getter_type=rdbms_custom,
-                                 setter_name='_server_update_set', setter_type=rdbms_custom, setter_arg_name='parameters',
-                                 custom_func_name='_server_update_custom_func')
-        g.generic_wait_command('wait', getter_name='_server_mysql_get', getter_type=rdbms_custom)
+        g.custom_command('create', '_server_create')
+        g.custom_command('georestore', '_server_georestore', no_wait_param='no_wait')
```

Comment:
To support no wait, we now support just doing `supports_no_wait=True`

See "(4) Supporting --no-wait" in https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands